src/main/java/com/starbucks/controllers/RestController.java

```diff
@@ -1,26 +1,38 @@
 package com.starbucks.controllers;
 
 import com.starbucks.model.Order;
-import com.starbucks.repositary.OrderRepositary;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
+import com.starbucks.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
 
 @org.springframework.web.bind.annotation.RestController
-@RequestMapping("/order")
 public class RestController {
 
-    private final OrderRepositary orderRepositary;
+    private final OrderService orderService;
 
-    public RestController(OrderRepositary orderRepositary) {
-        this.orderRepositary = orderRepositary;
+    public RestController(OrderService orderService) {
+        this.orderService = orderService;
     }
 
-    @PostMapping
+    @PostMapping("/orders")
     public void storeOrder(@RequestBody Order order) {
-        order.toString ( );
-        orderRepositary.saveOrder ( order );
+        orderService.saveOrder ( order );
     }
 
+    @PutMapping("/orders/{id}")
+    public void updateOrder(@RequestBody Order order , @PathVariable("id") UUID id) {
+        orderService.updateOrder ( order , id );
+    }
 
+    @DeleteMapping("/{id}")
+    public void deleteOrder(@PathVariable("id") UUID id) {
+        System.out.println ( id );
+        orderService.deleteOrdersById ( id );
+    }
+
+    @PatchMapping("/orders")
+    public void patchOrder(@RequestBody Order order) {
```

Comment:
Actually it's my fault, I should've told you that PATCH should use not Body but Parameter (like /orders?status=NEW) and you should have a separate method in the service for it probably.